routes/web.php

```diff
@@ -38,4 +41,10 @@
 Route::post('fish_records/delete/{id}','FishRecordController@delete')->name('delete');
 
 //釣り記録詳細ページ
-Route::get('fish_records/{id}','FishRecordController@detalis')->name('detalis');
\ No newline at end of file
+Route::get('fish_records/{id}','FishRecordController@detalis')->name('detalis');
+
+//釣り記録検索ページ
+Route::post('fish_records/search','FishRecordController@search')->name('search');
```

Comment:
データを変更しないような、取得するだけのルーティングは `get` が望ましいので修正しましょう🐱
```suggestion
Route::get('fish_records/search','FishRecordController@search')->name('search');
```